common/utils/filters.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  CatalogueAggregationBucket,
+  CatalogueAggregations,
+} from '@weco/common/model/catalogue';
+import { nodeListValueToArray, inputValue } from '@weco/common/utils/forms';
+import { LinkProps } from 'next/link';
+
+import {
+  worksLink,
+  imagesLink,
+} from '@weco/common/services/catalogue/ts_routes';
+
+export const getFilterByCount = (
+  bucketFilter: CatalogueAggregationBucket[]
+): CatalogueAggregationBucket[] => {
+  return bucketFilter.filter(items => {
+    return items.count > 0;
+  });
+};
+
+export const getAggregationFilterByName = (
+  aggregations: CatalogueAggregations | undefined,
```

Comment:
When would aggregations be `undefined`? We know that we get this from the API.

Reply:
Just looking back on this.. i would have to make aggregations a compulsory item to always return. Currently the aggregation is aggregations?: CatalogueAggregations; -> which makes me have to declare it undefined. I updated this to resolve the typechecking. 

Whats your thoughts on this @jamesgorrie. Should i change aggregations to   aggregations: CatalogueAggregations;

Reply:
Na, let's leave out that as it will have larger effects.